### PR TITLE
Added gta victy cheat sheats 

### PR DESCRIPTION
**Pull request by alfredmaliakal**
**What does your Instant Answer do?**
vice city game cheat

**What problem does your Instant Answer solve (Why is it better than organic links)?**
shortcuts for gta game

**What is the data source for your Instant Answer? (Provide a link if possible)**
http://www.gamefaqs.com/pc/561641-grand-theft-auto-vice-city/cheats

**Why did you choose this data source?**
valid source

**Are there any other alternative (better) data sources?**
na

**What are some example queries that trigger this Instant Answer?**
gta victy cheat codes, gta vice city cheat sheet

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
gamers

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
n/a

**Which existing Instant Answers will this one supercede/overlap with?**
n/a

**Are you having any problems? Do you need our help with anything?**
No need 

**Where did you hear about DuckDuckHack? (For first time contributors)**
Duckduckhack at CEK , Event at my college 

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

## Checklist
```
    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera
```

-----
IA Page: https://duck.co/ia/view/gta_vice_city_cheat_sheet

### CODE REVIEW

**Comment by gautamkrishnar** [comment]
#1611 duckdukhackcek

**Comment by gautamkrishnar** [comment]
@moollaza any idea why the build is failing

File: share/goodie/cheat_sheets/json/gta-vice-city-cheat-cheat.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
+{
+  "id": "gta_vice_city_cheat_sheet",
+  "name": "gta vice city",
+  "description": " GTA Vice City",
+  "metadata": {
+    "sourceName": "Game faqs",
+    "sourceUrl": "http://www.gamefaqs.com/pc/561641-grand-theft-auto-vice-city/cheats",
```
Comment (seisfeld) [inline share/goodie/cheat_sheets/json/gta-vice-city-cheat-cheat.json]: @gautamkrishnar You need to remove that comma at the end of line 7 to have travis passing.

Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/gta-vice-city-cheat-cheat.json]: Yeah, Thanks :+1:

**Comment by moollaza** [comment]
@alfredmaliakal Hey!

Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

File: share/goodie/cheat_sheets/json/gta-vice-city-cheat-cheat.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
+{
+  "id": "gta_vice_city_cheat_sheet",
+  "name": "gta vice city",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/gta-vice-city-cheat-cheat.json]: Let's use proper Title Casing here.

File: share/goodie/cheat_sheets/json/gta-vice-city-cheat-cheat.json [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  "description": " GTA Vice City",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/gta-vice-city-cheat-cheat.json]: Let's make this more description to actually describe what this cheat sheet shows, or if you think it's self-explanatory, we can remove the description altogether.

File: share/goodie/cheat_sheets/json/gta-vice-city-cheat-cheat.json [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  "metadata": {
+    "sourceName": "Game faqs",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/gta-vice-city-cheat-cheat.json]: Let's capitalize "Game Faqs"

File: share/goodie/cheat_sheets/json/gta-vice-city-cheat-cheat.json [hunk 5]
```diff
@@ -0,0 +1,73 @@
+{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/gta-vice-city-cheat-cheat.json]: whoops, your filename should just be "gta-vice-city.json"

File: share/goodie/cheat_sheets/json/gta-vice-city-cheat-cheat.json [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+  "id": "gta_vice_city_cheat_sheet",
+  "name": "gta vice city",
+  "description": " GTA Vice City",
+  "metadata": {
+    "sourceName": "Game faqs",
+    "sourceUrl": "http://www.gamefaqs.com/pc/561641-grand-theft-auto-vice-city/cheats",
+  },
+  "aliases": [
+    "gta cheat codes",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/gta-vice-city-cheat-cheat.json]: let's replace this one with "vice city". "gta" is a little too broad.

File: share/goodie/cheat_sheets/json/gta-vice-city-cheat-cheat.json [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+    "gta 2 cheats",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/gta-vice-city-cheat-cheat.json]: the aliases still require one of the trigger words ("cheat sheet", "cheatsheet", "reference", "help", etc) to appear in the query for them to work.

Some appropriate aliases for this Cheat Sheet would be: "vice city", "gta 2", "vice city cheats" so queries like "vice city cheat sheet", "gta 2 reference" or "vice city cheats help" would trigger this

**Comment by moollaza** [comment]
@alfredmaliakal Thanks a lot for contributing, we really appreciate it! I've left you some feedback that needs to be implemented. Once things are updated please let us know by leaving a comment, so we can review this again. If you have any questions, feel free to ask them here!

**Comment by gautamkrishnar** [comment]
@alfredmaliakal  You must also edit the pull request's title typo "victy"

**Comment by moollaza** [comment]
@alfredmaliakal Ping! I'm just checking in to see if you're still interested in finishing this one up? We'd love to see this go live!

There's absolutely no rush, just wanted to make sure you're not stuck on anything.

**Comment by moollaza** [comment]
@alfredmaliakal Ping! I'm just checking in again to see if you're still interested in finishing this one up? We'd love to see this go live!

There's absolutely no rush, just wanted to make sure you're not stuck on anything.

---

BTW, thanks for doing such a great job so far! :cake: We really appreciate you taking the time to contribute. As a thanks, after this is merged we'd like to write a short blog post about you and your Instant Answer.

If you're interested please send an email to open@duckduckgo.com, with the subject "DuckDuckHack Dev Appreciation".

Please provide your name and include a link to this pull request as well.

We'll get in touch for more details!

**Comment by gautamkrishnar** [comment]
@moollaza i will ping him now and remind him about this

**Pull request closed without merging**